_about/notes/0.5.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
+pod-level proxy injection. Automatic injection requires Kubernetes 1.9 or beyond and
+therefore doesn't work on older versions. [Learn more](https://istio.io/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection)
+
+## Mixer adapters
+
+- **Open Policy Agent**. Mixer now has an authorization adapter implementing the [open policy agent](http://www.openpolicyagent.org) model,
+providing a flexible fine-grained access control mechanism. TODO: Doc link?
+
+- **RBAC**. Mixer now has a resource-based access control adapter. TBD: Doc link?
+
+- **Fluentd**. Mixer now has an adapter for [fluentd](https://www.fluentd.org) telemetry collection. TODO: Doc link?
```

Comment:
telemetry -> log (adapter only accepts logentries)
Doc link: https://istio.io/docs/tasks/telemetry/fluentd.html
Not posted yet. https://github.com/istio/istio.github.io/pull/905